.github/workflows/setup.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Set up ruby
         uses: ruby/setup-ruby@v1
         with:
+          ruby-version: '3.0'
```

Comment:
note that latest is 3.2.2

Reply:
Thanks. It works, however the setup seems much slower

Reply:
Maybe not that much slower after all

Reply:
Sadly rubyXL doesn't seem to work with ruby 3.2.2 yet. So we need to stick with 3.0 for the time being...